study-builder/studies/pancan/family-history/family-history-top-level.conf

```diff
@@ -0,0 +1,484 @@
+    { "statusCode": "IN_PROGRESS", "language": "en", "text": ${i18n.en.family_history.status.inprogress} },
+    { "statusCode": "IN_PROGRESS", "language": "es", "text": ${i18n.es.family_history.status.inprogress} },
+    { "statusCode": "COMPLETE", "language": "en", "text": ${i18n.en.family_history.status.complete} }
+    { "statusCode": "COMPLETE", "language": "es", "text": ${i18n.es.family_history.status.complete} }
+  ],
+
+  "introduction": null,
+  "sections": [
+    {
+      "nameTemplate": {
+        "templateType": "TEXT",
+        "templateText": "$introduction",
+        "variables": [
+          {
+            "name": "introduction",
+            "translations": [
+              { "language": "en", "text": ${i18n.en.family_history.introduction} },
+              { "language": "es", "text": ${i18n.es.family_history.introduction} },
+            ]
+          }
+        ]
+      },
+      "icons": [],
+      "blocks": [
+        ${_includes.family_history.introduction}
+      ]
+    },
+    {
+      "nameTemplate": {
+        "templateType": "TEXT",
+        "templateText": "$instructions",
+        "variables": [
+          {
+            "name": "instructions",
+            "translations": [
+              { "language": "en", "text": ${i18n.en.family_history.instructions} },
+              { "language": "es", "text": ${i18n.es.family_history.instructions} },
+            ]
+          }
+        ]
+      },
+      "icons": [],
+      "blocks": [
+        ${INSTRUCTIONS_BLOCK_SELF},
+        ${INSTRUCTIONS_BLOCK_CHILD}
+      ]
+    },
+    {
+      "nameTemplate": {
+        "templateType": "TEXT",
+        "templateText": "$parents",
+        "variables": [
+          {
+            "name": "parents",
+            "translations": [
+              { "language": "en", "text": ${i18n.en.family_history.parents} },
+              { "language": "es", "text": ${i18n.es.family_history.parents} }
+            ]
+          }
+        ]
+      },
+      "icons": [],
+      "blocks": [
+        ${PARENT_SECTION_INTRO_BLOCK},
+        {
+          "activityCode": "FAMILY_HISTORY_PARENT1",
+          "blockType": "ACTIVITY",
+          "shownExpr": null,
+          "enabledExpr": """!user.studies["cmi-pancan"].forms["FAMILY_HISTORY"].questions["PARENT1_IDK"].answers.hasOption("DK")"""
+          "allowMultiple": false,
+          "renderHint": "MODAL"
+        },
+        {
+          "blockType": "QUESTION",
+          "question": {
+            "stableId": "PARENT1_IDK"
+            "questionType": "PICKLIST",
+            "selectMode": "MULTIPLE",
+            "renderMode": "LIST",
+            "isRestricted": false,
+            "isDeprecated": false,
+            "hideNumber": true,
+            "groups": [],
+            "validations": [],
+            "promptTemplate": {
+              "templateType": "HTML",
+              "templateText": "",
+              "variables": []
+            },
```

Comment:
What's going on here? It looks like this question is setup to have an empty prompt.

Reply:
That's correct! This block is just a single checkbox. The "I do not have any information..." here:
<img width="1195" alt="image" src="https://user-images.githubusercontent.com/71830706/142260133-cb30d34f-2fe7-4781-980c-5ba3f3e09197.png">

Reply:
In that case, the structure needs to be reworked. If the "I do not have any information..." text is intended to be presented with the checkbox, it should be included as part of the question's data. At the moment, there is no way to know these two elements should be presented together, and only happens to work because of the way the Angular client renders things.